helm-charts/Makefile

```diff
@@ -5,7 +5,8 @@ include ../Makefile.os
 RELEASE_VERSION ?= latest
 CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../release.version | tr A-Z a-z)
 CHART_NAME=strimzi-kafka-operator
-CHART_PATH=./$(CHART_NAME)/
+HELM_VERSION ?= helm3
```

Comment:
I wonder if the right solution here is to move the Makefile one level down and have a separate Makefile for Helm2 and sepaarte one for Helm 3? With some modification to the `SUBDIRS` in the main make file, that should also allow to build both helm charts in single run I assume.

Reply:
Yes, that was my initial thought as well, but I was trying to keep everything slightly more minimal. I will give that a go.